components/features/timeline/EventRegisterDrawer.tsx

```
@@ -61,10 +59,25 @@ export function EventRegisterDrawer({
     description: initialData?.description || "",
   })
 
+  const [selectedFile, setSelectedFile] = React.useState<File | null>(null)
   const [selectedImage, setSelectedImage] = React.useState<string | null>(
     initialData?.imageUrl || null
   )
 
+  // Drawerが開かれた際、および initialData が変わった際にフォームおよび画像選択状態を初期化する
+  React.useEffect(() => {
+    if (isOpen) {
+      setFormData({
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/components/features/timeline/EventRegisterDrawer.tsx:70:7
  68 |   React.useEffect(() => {
  69 |     if (isOpen) {
> 70 |       setFormData({
     |       ^^^^^^^^^^^ Avoid calling setState() directly within an effect
  71 |         title: initialData?.title || "",
  72 |         year: initialData?.year || "2024",
  73 |         month: initialData?.month || "1",

(react-hooks/set-state-in-effect)